numba/core/config.py

```diff
@@ -135,7 +135,11 @@ def validate(self):
                 CUDA_USE_NVIDIA_BINDING = False
 
             if CUDA_PER_THREAD_DEFAULT_STREAM:  # noqa: F821
-                warnings.warn("PTDS is not supported with CUDA Python")
+                warnings.warn("PTDS support is handled by CUDA Python when "
+                              "using the NVIDIA binding. Please set the "
+                              "environment variable "
+                              "CUDA_PYTHON_CUDA_PER_THREAD_DEFAULT_STREAM to 1 "
+                              "instead.")
```

Comment:
I've manually tested that this warning appears correctly. Two questions 1. is it worth adding a test for it to the test suite 2. Should it be a warning of class `RuntimeWarning`?

Reply:
1. This will be temporary, PTDS is presently a pretty niche use case, and I don't expect the code around this to change much, so I was comfortable without the test - my estimation was that the time saved by adding it wouldn't exceed the time consumed implementing it. I can add one if you would really like one though.
2. Most of the other warnings in `config.py` are not - is there a reason it should be?

Reply:
1. Thanks, I agree RE effort given the longevity of the change and have manually tested too. These comments record the reasoning :)
2. I don't think so, was just wondering if you had a preference, let's leave as-is. The warning classes are a little inconsistent in `config.py` and should probably be all the same/something special to Numba to permit filtering.